drgn_tools/runq.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) 2023, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/
+import argparse
+
+from drgn import container_of
+from drgn import Object
+from drgn import Program
+from drgn.helpers.common import escape_ascii_string
+from drgn.helpers.linux.cpumask import for_each_online_cpu
+from drgn.helpers.linux.list import list_for_each_entry
+from drgn.helpers.linux.percpu import per_cpu
+
+from drgn_tools.corelens import CorelensModule
+
+# List runqueus per cpu
+
+
+def print_rt_runq(runqueue: Object) -> None:
+    """ """
+    count = 0
+    prio_array = hex(runqueue.rt.active.address_ - 16)
+    print("\n \t RT PRIO_ARRAY:", prio_array)
+    rt_prio_array = runqueue.rt.active.queue
+    for que in rt_prio_array:
+        for t in list_for_each_entry(
+            "struct sched_rt_entity", que.address_of_(), "run_list"
+        ):
+            count += 1
+            tsk = container_of(t, "struct task_struct", "rt")
+            if tsk.pid:
+                print(
+                    "[{}]\tPID: {}\tTASK: {}\tCOMMAND: {}".format(
+                        tsk.prio.value_(),
+                        tsk.pid.value_(),
+                        hex(tsk),
+                        escape_ascii_string(tsk.comm.string_()),
+                    )
+                )
+            else:
+                print()
+    if count == 0:
+        print(" \t\t[no tasks queued]")
+
+
+def print_cfs_runq(runqueue: Object) -> None:
+    """ """
+    cfs_root = hex(runqueue.cfs.tasks_timeline.address_of_().value_())
+    print("\n \t CFS RB_ROOT:", cfs_root)
+    count = 0
+    runq = runqueue.address_of_()
+    for t in list_for_each_entry(
+        "struct task_struct", runq.cfs_tasks.address_of_(), "se.group_node"
+    ):
+        count += 1
+        print(
+            "[{}]\tPID: {}\tTASK: {}\tCOMMAND: {}".format(
```

Comment:
Similar to the comments on the rt prio array for printing each task:
```suggestion
            '     [{:3d}] PID: {:<6d} TASK: {}  COMMAND: "{}"'.format(
```